tests/e2e/helpers.go

```diff
@@ -1565,3 +1576,24 @@ func runPod(t *testing.T, f *framework.Framework, namespace string, podToRun *co
 func createAndRemoveEtcSecurettyOnNode(t *testing.T, f *framework.Framework, namespace, name, nodeName string) (*corev1.Pod, error) {
 	return runPod(t, f, namespace, createAndRemoveEtcSecurettyPod(namespace, name, nodeName))
 }
+
+func createReadyProfileBundle(t *testing.T, f *framework.Framework, name, namespace, contentImage, contentFile string) error {
```

Comment:
instead of this, what about already creating some profile bundles by default?

Reply:
You mean the test suite itself would create the bundles?

Reply:
I mean that, when the operator starts, we could create some bundles by default. Let me give it a try,